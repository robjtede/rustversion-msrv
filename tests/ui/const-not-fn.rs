#[rustversion_msrv::attr(all(), const)]
pub struct S;

fn main() {}
