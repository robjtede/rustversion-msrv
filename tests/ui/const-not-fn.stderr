error: only allowed on a fn item
 --> tests/ui/const-not-fn.rs:1:33
  |
1 | #[rustversion_msrv::attr(all(), const)]
  |                                 ^^^^^
